### DISCUSSION TITLE
Moving smoothly in a tile based game?

### DISCUSSION THREAD

**Discussion by cowmisty** [avianphysics/avian#605] [Q&A]
Making a tile based platformer game, each tile has their rectangular collider. When moving above them, I feel stuttering. I tried setting gravityscale to 0 when on ground but doesn't seem to help. I compared it to moving on a single long collider and it is smoother. Should I detect continuous sides and replace them with one collider, or is there another way? What is recommended way? (I am using a dynamic body).

**Comment by Chefaku** [comment]
I have the same issue, also if you use a rectangle as the player, it gets stuck between the tiles. Maybe a step handling system could solve it
